### PR TITLE
Check Google Play service is available, if not fallback 

### PR DESCRIPTION
**Pull request by jigar-f**

### CODE REVIEW

File: android/app/src/main/kotlin/org/getlantern/lantern/model/InAppBilling.kt [hunk 1]
```diff
@@ -54,6 +56,15 @@ class InAppBilling(
     @Volatile
     private var purchasesUpdated: PurchasesUpdatedListener? = null
 
+
+    fun isPlayStoreAvailable(): Boolean {
+        if (billingResult != null) {
```
Comment (atavism) [inline android/app/src/main/kotlin/org/getlantern/lantern/model/InAppBilling.kt:61]: Beyond checking the billing client is initialized, does it make sense to use something like [GoogleApiAvailability.isGooglePlayServicesAvailable()](https://developers.google.com/android/reference/com/google/android/gms/common/GoogleApiAvailability#isGooglePlayServicesAvailable%28android.content.Context%29) here to check that Google Play services is installed and enabled on the device?

**Comment by atavism** [comment]
LGTM, just one comment

File: android/app/src/main/kotlin/org/getlantern/lantern/model/InAppBilling.kt (hunk 1 above)
Comment (jigar-f) [reply android/app/src/main/kotlin/org/getlantern/lantern/model/InAppBilling.kt:61]: According to this-:https://stackoverflow.com/questions/52858403/check-if-in-app-billing-via-playstore-is-available-to-user, billingResult will work just fine.

File: desktop/lib.go [hunk 2]
```diff
@@ -85,7 +85,7 @@ func start() {
 	settings := loadSettings(cdir)
 	webclientOpts := &webclient.Opts{
 		HttpClient: &http.Client{
-			Transport: proxied.ParallelPreferChained(),
+			Transport: proxied.ChainedThenFronted(),
```
Comment (jigar-f) [inline desktop/lib.go:88]: @atavism I had changed this due to an error, You can check Slack more about that, For now even using proxied.ChainedThenFronted gives error on the first request.

**Comment by jigar-f** [comment]
@atavism is all good here?, Should I go ahead and merge this?

**Comment by atavism** [comment]
@jigar-f Yep, this one LGTM. Merging...

File: android/app/src/main/kotlin/org/getlantern/lantern/model/InAppBilling.kt (hunk 1 above)
Comment (atavism) [reply android/app/src/main/kotlin/org/getlantern/lantern/model/InAppBilling.kt:61]: Sounds good!

**Pull request merged**